cfpb/consumerfinance.gov
# Archiving: add archive pill and image grayscale

Adds archive pill and image grayscale filter on pages that have been marked as archived.


## Additions

- Archive pill
- Archive class for archived pages
- Grayscale filter for images on archived pages (only works on modern browsers / Edge)



## How to test this PR

1. Check out branch
2. Run gulp styles
3. Open [a blog page](http://localhost:8000/admin/pages/1492/edit/#tab-configuration) in admin and mark it as archived
4. Navigate to page and verify that archive pill appears above header and images are grayscale
5. Repeat previous 2 steps for newsroom and research & reports


## Screenshots

<img width="801" alt="Screen Shot 2020-09-25 at 10 47 11 AM" src="https://user-images.githubusercontent.com/778171/94300087-5b5a4b00-ff1d-11ea-94eb-93c78bc7711d.png">



## Checklist


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".


### Front-end testing

#### Browser testing

Visually tested in the following supported browsers:
